src/bundle/Resources/views/admin/content_type/edit.html.twig

```diff
@@ -44,7 +44,7 @@
     <section class="container mt-4 px-5">
         <div class="card ez-card">
             <div class="ez-card__header ez-card__header--secondary d-flex justify-content-between">
-                <div class="p-2">
+                <div class="title p-2">
```

Comment:
I think `.title` is not present in bootstrap so you should use our class convention. This could something like: `.ez-card__title`